### PR TITLE
Code Quality: Migrated `GetCursorPos` to CsWin32

### PR DESCRIPTION
**Pull request by gumbarros**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Part of #14505, #15000 

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

File: src/Files.App/NativeMethods.txt [hunk 1]
```diff
@@ -42,4 +42,5 @@ SetActiveWindow
 CopyFileFromApp
 MoveFileFromApp
 DeleteFileFromApp
-RemoveDirectoryFromApp
\ No newline at end of file
+RemoveDirectoryFromApp
+GetCursorPos
```
Comment (0x5bfa) [inline src/Files.App/NativeMethods.txt:46]: I think we have one already in line 16.

Comment (gumbarros) [reply src/Files.App/NativeMethods.txt]: Done

**Comment by gumbarros** [comment]
Sorry, I am on the phone and probably messed the license at NativeMethods

File: src/Files.App/NativeMethods.txt [hunk 2]
```diff
@@ -1,5 +1,4 @@
-// Copyright (c) 2024 Files Community
-// Licensed under the MIT License. See the LICENSE.
+// Copyright (c) 2024 Files Community// Licensed under the MIT License. See the LICENSE.
```
Comment (yaira2) [inline src/Files.App/NativeMethods.txt:1]:
```suggestion
// Copyright (c) 2024 Files Community
// Licensed under the MIT License. See the LICENSE.
```

**Review by 0x5bfa — APPROVED** [review]

**Pull request merged**